src/DroneSafetyApi/Startup.cs

```diff
@@ -53,9 +53,15 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddMvc();
 
-            //services.Configure<DbOptions>(Configuration.GetSection("DocumentDB"));
-            //services.AddSingleton<IHazardRepository, HazardDatabase>();
-            services.AddSingleton<IHazardRepository, ExampleHazardRepository>();
+            services.Configure<DbOptions>(Configuration);
+            if (Configuration.GetValue<string>("EndpointUri") == null)
+            {
+                services.AddSingleton<IHazardRepository, ExampleHazardRepository>();
+            }
+            else
+            {
+                services.AddSingleton<IHazardRepository, HazardDatabase>();
+            }
```

Comment:
As in you have linked up a docdb database and queried something and a sensible looking output was returned on the web app